wiki/Article_styling_criteria/Writing/en.md

```diff
@@ -0,0 +1,343 @@
+
+British English spelling variants must be used when there are conflicting spellings of English words. For example, use:
+
+- `colour` instead of `color`
+- `centre` instead of `center`
+- `skilful` instead of `skillful`
+- `analyse` instead of `analyze`
+
+## Capitalisation
+
+### Article names
+
+Article names, when written in a paragraph, must not be capitalised, unless it is a link that links to the article or it is a proper noun (e.g. BanchoBot). The following is an example:
+
+```markdown
+The [Chat Console](/wiki/Chat_Console) is where you go to chat, using IRC (Internet Relay Chat). The chat console is located in the game client.
+```
+
+### Game modifiers
+
+Game modifiers must have the word `mod` after the mod name to reduce ambiguity. These must be spelt as seen below (letter casing and spacing must match):
+
+- `Easy` or `EZ`
+- `No Fail` or `NF`
+- `Half Time` or `HT`
+- `Daycore` or `DC`
+- `Hard Rock` or `HR`
+- `Sudden Death` or `SD`
+- `Perfect` or `PF`
+- `Double Time` or `DT`
+- `Nightcore` or `NC`
+- `Fade In` or `FI`
+- `Hidden` or `HD`
+- `Flashlight` or `FL`
+- `Relax` or `RL`
+- `Autopilot` or `AP`
+- `Target Practice` or `TP`
+- `Spun Out` or `SO`
+- `1K`, `2K`, `3K`, `4K`, `5K`, `6K`, `7K`, `8K`, and `9K`
+  - If referring to the key mods collectively, use `xK`.
+- `Co-op` or `CO`
+- `Random` or `RD`
+- `Mirror` or `MR`
+- `Auto` or `AT`
+- `Cinema` or `CM`
+- `Touch Device` or `TD`
+- `ScoreV2`
+
+---
+
+These game modifiers are no longer being used by osu!; however, if needed, these must be spelt as seen below (letter casing and spacing must match):
+
+- `Fade Out`
+- `No Video`
+- `10K`
+- `Taiko`
+
+---
+
+When writing out game modifiers for tournament articles, they must instead use upper camel case (omit the space and keep the letter casing as seen above).
+
+### Gameplay elements
+
+Gameplay elements must not be capitalised, unless it is a link that links to the article. The following is an example:
+
+```markdown
+In osu!standard, beatmaps are composed of three different gameplay elements: hit circles, sliders, and spinners.
+```
+
+### Language names
+
+Language names must be capitalised. The following is an example:
+
+```markdown
+The `#spanish` chat channel is for those who speak Spanish.
+```
+
+### Proper nouns
+
+Proper nouns must be capitalised. The following is an example:
+
+```markdown
+Dean Herbert (also known as peppy) created osu! back in 2007.
+```
+
+### Trademarks
+
+*For rules regarding osu!, see: [osu!](#osu!)*
+
+The following trademarks must be spelt as follows (letter casing must match):
+
+- `Discord`
+- `Facebook`
+- `GitHub`
+- `Google`
+- `Reddit`
+- `Skype`
+- `Twitch`
+- `Twitter`
+- `YouTube`
+
+Trademarks must not be followed by the trademark or registered trademark symbols.
+
+## Date and time
+
+### Date formatting
+
+*For date formatting, see [Wikipedia's Manual of Style on Date formatting](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Dates_and_numbers#Formats)*
```

Comment:
while it's a good reference, it's completely useless in the sense it gives you zero information on what format should be chosen, or is preferable. is it 2020-01-01, or Jan 1st, 2020, or 1 Jan 2020?

Reply:
Could have probably phrased it better... To answer, there is actually no single preferred format for dates currently. The ones you listed are all acceptable. Though, I believe that the `2020-01-01` format should only be used in tables, while the other two should be used in paragraphs while probably keeping it consistent within the article.

I did define a date format in the earlier versions of ASC, but when I was updating the date formatting section a few PR's ago, clayton (or Joehuu?) suggested to link to that Wikipedia MoS section to keep it simple.

Reply:
even outlining one of these formats as a preferable one is already a start

Reply:
Maybe this if we're going to stick with it?
```suggestion
The formats found in [Wikipedia:Manual of Style/Dates and numbers § Formats](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Dates_and_numbers#Formats "Wikipedia") are all accepted date formats.
```

Reply:
yeah, that works

Reply:
when writing new stuff i prefer 1 Jan 2020 because i thought that was the british english way to do it. might be pulling that out of my ass though cuz i barely know the difference between american and british english beyond coloUr